subcontracted_service_product/models/procurement_group.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+from odoo.exceptions import UserError
+
+
+class ProcurementGroup(models.Model):
+    _inherit = "procurement.group"
+
+    @api.model
+    def _skip_procurement(self, procurement):
+        res = super()._skip_procurement(procurement)
+        if self._is_subcontracted_service_procurement(procurement.product_id):
+            return False
+        return res
+
+    @api.model
+    def _has_subcontracted_service_procurement(self, product_id):
+        return product_id.type == "product" and product_id.subcontracted_product_id
+
+    @api.model
+    def _is_subcontracted_service_procurement(self, product_id):
+        return (
+            product_id.type == "service"
+            and len(
+                self.env["product.template"].search(
+                    [("subcontracted_product_id", "=", product_id.id)]
+                )
+            )
+            > 0
+        )
+
+    @api.model
+    def _get_rule(self, product_id, location_id, values):
+        res = super()._get_rule(product_id, location_id, values)
+        if self._has_subcontracted_service_procurement(product_id):
+            self.trigger_service_procurement(product_id, location_id, values)
+        return res
```

Comment:
wouldn't it be more correct to do it on the _run_buy method?

Reply:
Because this is needed when creating a MTS sales order it has to be that way.